calligraphy/src/main/java/io/github/inflationx/calligraphy3/Calligraphy.java

```diff
@@ -3,23 +3,27 @@
 import android.annotation.SuppressLint;
 import android.annotation.TargetApi;
 import android.content.Context;
+import android.content.res.Resources;
 import android.graphics.Typeface;
 import android.os.Build;
 import android.text.TextUtils;
 import android.util.AttributeSet;
+import android.util.Log;
 import android.view.View;
 import android.view.ViewTreeObserver;
 import android.widget.TextView;
 
 import java.lang.ref.WeakReference;
 import java.lang.reflect.Method;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.widget.Toolbar;
 
 class Calligraphy {
 
     private static final String ACTION_BAR_TITLE = "action_bar_title";
     private static final String ACTION_BAR_SUBTITLE = "action_bar_subtitle";
+    private static final String TAG = Calligraphy.class.getSimpleName();
 
     /**
      * Some styles are in sub styles, such as actionBarTextStyle etc..
```

Comment:
``` Suggestion: comment in only one line ```

Reply:
@lucalves I am not sure why you suggesting this change here. Javadoc has not changed and it's not part of this pull request. Won't change.